.codeclimate.yml

```diff
@@ -10,4 +10,4 @@ exclude_paths:
 - "**/*.md"
 - "Dockerfile"
 - "bin/fixme"
-- "tests/**"
+- "test/**"
```

Comment:
I think the thinking was that specs aren't as as high-value, and that there are often patterns you *want* (or at least accept) in specs that you don't elsewhere (like longer methods & duplicated bits). The cost of avoiding these things or ignoring the issues they would result in is higher than the value of being told you forgot a comma.

That said I agree that it's weird that we tend to end up doing the basic listing work ourselves. That's why we built the bloody robots, after all.

Possibly we could compromise by excluding tests only from certain engines (like duplication), and using folder-level config for engines that support it (like rubocop) to apply a looser set of rules to specs.